tests/LanguageServerRename/Unit/Model/Renamer/ChainRenamerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerRename\Tests\Unit\Model\Renamer;
+
+use PHPUnit\Framework\TestCase;
+use Phpactor\Extension\LanguageServerRename\Model\Renamer\ChainRenamer;
+use Phpactor\Extension\LanguageServerRename\Model\RenameResult;
+use Phpactor\Extension\LanguageServerRename\Model\Renamer\InMemoryRenamer;
+use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\ByteOffsetRange;
+use Phpactor\TextDocument\TextDocumentBuilder;
+use Phpactor\TextDocument\TextDocumentUri;
+use Phpactor\TextDocument\TextEdits;
+use function iterator_to_array;
+
+class ChainRenamerTest extends TestCase
```

Comment:
I simplified this test a bit -- normally I'd actually use mocks here as it's part of the model. This test currently does not test f.e. that the `$newName` was passed to the `Renamer` but I can live with that.